woocommerce/woocommerce-gutenberg-products-block
# Product Image block: Enable global style

~This PR is blocked by #5542~

This PR adds support for global style for the `Product Image block`.

Now, the user can edit the style for:
- border-radius
- font-size

#4965 

### Screenshots

![image](https://user-images.githubusercontent.com/4463174/149410731-b9370a5a-29ec-4ab3-a8ea-11579945945c.png)
*without global style*


![image](https://user-images.githubusercontent.com/4463174/149410030-1675fa4f-c7ad-4147-bacb-6db1f4ead431.png)
*with global style*


### Testing

### Manual Testing

Check out this branch:

1. On `WordPress 5.9`, install and enable the `Gutenberg` plugin.
2. Install and enable the `Twenty Twenty-Two` theme.
3. Add the `Single Product block` (this block contains `Product Image Block`) to a post.
4. Get the focus on `Product Image Block`.
5. On the right sidebar, personalize the styles of the block.
6. Go on the page and check if there are changes.
7. Reset to default using the `Reset` button from the different sections. 
8. Go to Dashboard and select Appearance > Editor (beta). On top of the screen, select Home > Browser all templates > Single Post. When the page is loaded, add the block to the page.
9. On the Editor page click on the `Styles` icon on the right-top corner.
10. Verify that the `Product Image block` is shown under the `Blocks` section. Personalize again the block.
11. Save your changes.
12. Go on the page created earlier and check if all styles are applied correctly.
13. Edit your previous post/page again.
14. Change again the styles.
15. Save your changes.
16. Check if these styles have priority over the styles from the Site Editor.

### Changelog

> Add support for the global style for the Product Image block.
